service/status-report/main.go

```diff
@@ -215,6 +222,26 @@ func setClusterInformation(ctx context.Context, cl client.Client, status interfa
 
 }
 
+func setStorageQuotaUtilizationRatio(ctx context.Context, cl client.Client, status interfaces.StorageClientStatus) {
+	clusterResourceQuota := &quotav1.ClusterResourceQuota{}
+	clusterResourceQuota.Name = status.GetClientName()
+
+	// No need to check for NotFound because unlimited quota client will not have CRQ resource
+	if err := cl.Get(ctx, client.ObjectKeyFromObject(clusterResourceQuota), clusterResourceQuota); client.IgnoreNotFound(err) != nil {
+		klog.Warningf("Failed to get clusterResourceQuota %q: %v", clusterResourceQuota.Name, err)
+	}
+
+	if clusterResourceQuota.Status.Total.Hard != nil {
+		total := clusterResourceQuota.Status.Total.Hard["requests.storage"]
+		if total.AsApproximateFloat64() > 0 {
+			used := clusterResourceQuota.Status.Total.Used["requests.storage"]
```

Comment:
Hard and Used are of type `map[string]corev1.ResourceQuantity` this means that if the key `requests.storage` is not defined in either of the maps you will get a default `corev1.ResourceQuantity` which points to a value of 0. But that does not mean the utilization is 0, it just means that no controller has set this value. 

```suggestion
		total, totalExists := clusterResourceQuota.Status.Total.Hard["requests.storage"]
		used, usedExists := clusterResourceQuota.Status.Total.Used["requests.storage"]
		if totalExists && usedExists && total.AsApproximateFloat64() > 0 {
```